### ISSUE TITLE
Setup CI/tests

### ISSUE THREAD

**Issue by Enigmage** [ankingcodes/codemon#44]
Opening this on the request of @ankingcodes . It would be helpful to have an automated CI system that can test all PRs for proper coding style (proper indentation and other python standards) and merge/reject them based on the results.

**Comment by lucifer0987** [comment]
@Enigmage Are you working on this one? I am asking since you have created this issue and haven't specified if you are working or not. If you are not working, i would like to work on it.

**Comment by Enigmage** [comment]
@lucifer0987 Yeah sure go ahead. I'm not expecting this one to be only assigned to myself. Anyone can work on adding tests as mentioned in #45.

**Comment by ankushbhardwxj** [comment]
@lucifer0987 @Enigmage - The tests are independent to the CI, the CI will just run all the tests on  that branch once a PR is made.

**Comment by paramsiddharth** [comment]
The issue has been free for a couple of weeks now. May I work on the CI and tests?

**Comment by ankushbhardwxj** [comment]
@paramsiddharth - before setting up the CI, please make a PR with the tests only.
